cmd/daytona/config/const.go

```diff
@@ -65,6 +66,8 @@ func GetDocsLinkFromGitProvider(providerId string) string {
 		return "https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token"
 	case "bitbucket":
 		return "https://support.atlassian.com/bitbucket-cloud/docs/create-an-app-password"
+	case "bitbucket-server":
+		return "https://confluence.atlassian.com/bitbucketserver/using-ssh-keys-to-secure-git-operations-776639772.html"
```

Comment:
This link should be https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html

Reply:
replaced.